### PR TITLE
Add rule for other media types than json

### PR DESCRIPTION
**Pull request by fmueller**
As discussed in #11 this adds another rule for other media types than json if more appropriate for a given use case.

### CODE REVIEW

**Comment by tkrop** [comment]
It will have a small merge conflict with #63, but please go on to merge.

**Comment by tkrop** [comment]
:+1:

File: data-formats/DataFormats.md [hunk 1]
```diff
@@ -3,7 +3,13 @@
...
 
 JSON-encode the body payload. The JSON payload must follow [RFC-7159](https://tools.ietf.org/html/rfc7159) by having
-(if possible) a serialized object or array as the top-level object.
+(if possible) a serialized object or array as the top-level object. Please see our [JSON Guidelines](../json-guidelines/JsonGuidelines.md) for
+more guidance.
+
+## {{ book.could }} Use other Media Types than JSON
+
+If for given use case JSON does not make sense, for instance when providing attachments in form of PDFs, you should
+use another, more sufficient media type. But only do this if you can not transfer the information in JSON.
```
Comment (ePaul) [inline data-formats/DataFormats.md]: I think the wording is a bit ugly, but I currently can't think of how to make it better.

(The content is fine.)

Comment (fmueller) [inline data-formats/DataFormats.md]: @dehora Can you improve the wording as native speaker please? :-)

**Comment by fmueller** [comment]
@ePaul Are you ok with the content and would like to upvote?

**Comment by ePaul** [comment]
:+1:

**Comment by fmueller** [comment]
@tkrop @ePaul Please upvote again, I just merged master branch.

**Comment by whiskeysierra** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Pull request merged**